tools/package.json

```diff
@@ -37,6 +37,7 @@
     "open": "0.0.5",
     "request": "2.74.0",
     "requirejs": "2.2.0",
+    "rimraf": "2.5.4",
```

Comment:
No need to include this, if you need to remove files you can use `fsExtra.remove` instead.